tests/beta/tapes/test_tape.py

```diff
@@ -1507,3 +1503,46 @@ def test_trainable_measurement(self, tol):
         res = tape.jacobian(dev)
         expected = np.array([[-2 * a * np.sin(phi)]])
         assert np.allclose(res, expected, atol=tol, rtol=0)
+
+
+class TestGetAllParameters:
+    """Tests for the _get_all_parameters method"""
+
+    def test_all_params_trainable(self):
+        """Test that the input params are simply returned when all params are trainable"""
+        with QuantumTape() as tape:
+            qml.RX(0.1, wires=0)
+            qml.RX(0.2, wires=1)
+            qml.CNOT(wires=[0, 1])
+            expval(qml.PauliZ(wires=1))
+
+        new_params = tape._get_all_parameters([0.8, 0.9])
```

Comment:
`new_params` sounds like a good name for the original argument too in `_get_all_parameters` :slightly_smiling_face: